simapp/simd/cmd/testnet.go

```diff
@@ -247,7 +247,7 @@ func initTestnetFiles(
 			return err
 		}
 
-		memo := fmt.Sprintf("%s@%s:26656", nodeIDs[i], ip)
+		memo := fmt.Sprintf("%s@%s:%s", nodeIDs[i], ip, p2pURL.Port())
```

Comment:
What is `p2pURL` here? 26656 should work. We don't generate random ports, do we?